vestings/csv_parser.py

```diff
@@ -33,20 +33,23 @@ def parse_vestings_csv(db: orm.Session, type, chain_id, verbose):
 
             start_date: int
             if "startDate" in row.keys():
-                start_date = int(datetime.datetime.strptime(row["startDate"], "%Y-%m-%dT%H:%M:%S%z").timestamp())
+                start_date = parse(row["startDate"]).timestamp()
             else:
-                start_date = int(datetime.datetime.strptime("2018-09-27T10:00:00+00:00", "%Y-%m-%dT%H:%M:%S%z").timestamp())
+                start_date = parse("2018-09-27T10:00:00+00:00").timestamp()
 
             amount = row["amount"]
-            curve_type = 0
-
-            vesting = Vesting(None, type, owner, curve_type, duration_weeks, start_date, amount, None)
 
-            USER_AIRDROP_ADDRESS = MAINNET_USER_AIRDROP_ADDRESS if chain_id == 1 else RINKEBY_USER_AIRDROP_ADDRESS
-            ECOSYSTEM_AIRDROP_ADDRESS = MAINNET_ECOSYSTEM_AIRDROP_ADDRESS if chain_id == 1 else RINKEBY_ECOSYSTEM_AIRDROP_ADDRESS
+            curve_type = 0
 
-            vesting_id = vesting.calculateHash(USER_AIRDROP_ADDRESS, chain_id) if type == "user" \
-                else vesting.calculateHash(ECOSYSTEM_AIRDROP_ADDRESS, chain_id)
+            vesting_id: str
+            if "vestingId" in row.keys():
+                vesting_id = row["vestingId"]
```

Comment:
I would not blindly trust the vesting id. A better logic would be to always calculate it and if it is set in the csv to compare it to the calculated one. If it is different then throw an error.